Update binary stream type

filippog opened the pull request:
also a small fix to Makefile to DTRT when changing source and running `make test` right after

armon commented:
@filippog Could you simplify the diff to remove the unnecessary changes? It seems there should be:
* A single new define value
* A new line of STREAM_BIN
* Updated README with new value
* Updated tests for new binary stream value

The changes to makefile and the other lines shouldn't be in here.

filippog commented:
@armon yep, I've updated this PR and split the Makefile change in #126

the reordering of lines was to keep the same order in ascii and binary stream, are they supposed to match in what stats are exported?

armon commented:
Thanks!
